docs/guides/connecting_to_your_data/in_memory/spark.md

```diff
@@ -88,14 +87,12 @@ Save the configuration into your `DataContext` by using the `add_datasource()` f
   ]}>
 
 <TabItem value="yaml">
-
-```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_yaml_example.py#L46
+```python name="tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_yaml_example.py add datasource"
 ```
 
 </TabItem>
 <TabItem value="python">
-
-```python file=../../../../tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_python_example.py#L46
+```python name="tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_python_example.py add datasource"
```

Comment:
For some reason these are not rendering: 
![image](https://user-images.githubusercontent.com/9903066/220690432-148dcd08-afd4-4067-8659-04fdd09e6717.png)

Reply:
I think there was some tab-related issue with this doc. Will check it out